_skycap/the-basics/formations-stencils-and-snapshots.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+template: one-col
+title: Formations, stencils and snapshots
+categories: the-basics
+lead: "Understand the terminology specific to Skycap"
+legacy: false
+tags: ["the basics","concepts", "terminology","explanation"]
+permalink: /:collection/:path
+---
+
+## Overview
+
+Kubernetes provides amazing flexibilty to developers and operators. By consolidating all communication through a single API it encourages developing code to build, configure and maintain infrastructure. This brings great benefits but also some challenges. 
+
+As your application evolves, so do the configuration files that are required to run it on the infrastructure: new components like Redis or Memcached are added and old ones removed, application configurations and secrets change and the number of micro-services keeps growing. This means you need to keep your configuration files not only flexible and managable, but also version controlled.
+
+Storing your configuration files in your source code, next to the rest of the code is one option. But what if your application is not store in a single git repository (mono-repo) and is developed and controlled by different teams? How do you ensure configuration changes that are required by developers of a certain service are in lock-step with the code? 
```

Comment:
`application is not *store*` store should be stored?

Reply:
Oops, thanks. Have fixed.